From: Egbert Eich <eich@suse.com>
Date: Wed Oct 6 14:30:12 2021 +0200
Subject: provision: Unify handling of initramfs location
Patch-mainline: Not yet
Git-commit: 6884063cbbafab0e166ae4375f9ad608dbedd933
References: 

warewulf/ipmi derives the initramfs location from
WW_STATEDIR which is set in warewulf/common.

Signed-off-by: Egbert Eich <eich@suse.com>
Signed-off-by: Egbert Eich <eich@suse.de>
---
 provision/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
diff --git a/provision/configure.ac b/provision/configure.ac
index 4ee0c64..8deb735 100644
--- a/provision/configure.ac
+++ b/provision/configure.ac
@@ -161,7 +161,7 @@ AC_SUBST(fulllocalstatedir)
 AC_EXPAND_DIR(fulldatadir, "$datadir")
 AC_SUBST(fulldatadir)
 
-initramfsdir="$localstatedir/warewulf/initramfs/";
+initramfsdir="$WAREWULFSTATEDIR/warewulf/initramfs/";
 AC_SUBST(initramfsdir)
 
 BUSYBOX_CONFIG_EXTRA_LDLIBS=
